### PR TITLE
Add how-to for migrating from openshift4-cloudscale

### PR DESCRIPTION
**Pull request by simu**
## Checklist

- [x] Keep pull requests small so they can be easily reviewed.
- [x] Update the documentation.
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog

### CODE REVIEW

File: docs/modules/ROOT/pages/how-tos/migrate-from-openshift4-cloudscale.adoc [hunk 1]
```diff
@@ -0,0 +1,93 @@
+= Migrate from component `openshift4-cloudscale`
+
+This component supports both the Terraform module for OpenShift 4 on Exoscale and cloudscale.ch.
+We previously had a dedicated component `openshift4-cloudscale` for driving the Terraform module for OpenShift 4 on cloudscale.ch.
+That component is deprecated in favor of this component.
+
+This how-to provides a guide for migrating from component `openshift4-cloudscale` to `openshift4-terraform`.
+
+We recommend that you first perform the component migration and then follow the how-to for xref:how-tos/upgrade-cloudscale-v1-v2.adoc[upgrading the cloudscale.ch Terraform module from v1 to v2].
```
Comment (ccremer) [inline docs/modules/ROOT/pages/how-tos/migrate-from-openshift4-cloudscale.adoc:9]:
```suggestion
Previously there was a dedicated component `openshift4-cloudscale` for driving the Terraform module for OpenShift 4 on cloudscale.ch.
That component is deprecated in favor of this component.

This how-to provides a guide for migrating from component `openshift4-cloudscale` to `openshift4-terraform`.

It's highly recommended that you first perform the component migration and then follow the guide to xref:how-tos/upgrade-cloudscale-v1-v2.adoc[upgrade the cloudscale.ch Terraform module from v1 to v2].
```

File: docs/modules/ROOT/pages/how-tos/migrate-from-openshift4-cloudscale.adoc [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+== Prerequisites
+
+* Todo
```
Comment (ccremer) [inline docs/modules/ROOT/pages/how-tos/migrate-from-openshift4-cloudscale.adoc:13]: Todo :)

File: docs/modules/ROOT/pages/how-tos/migrate-from-openshift4-cloudscale.adoc [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
++
+.Move parameters from openshift4_cloudscale to openshift4_terraform
+[source,bash]
+----
+yq eval -i '.parameters.openshift4_terraform = .parameters.openshift4_cloudscale |
+  del(.parameters.openshift4_cloudscale) |
+  .parameters.openshift4_terraform.terraform_variables =
+    .parameters.openshift4_terraform.variables |
+  del(.parameters.openshift4_terraform.variables)' "${CLUSTER_ID}.yml"
+----
```
Comment (ccremer) [inline docs/modules/ROOT/pages/how-tos/migrate-from-openshift4-cloudscale.adoc:68]: Could be combined into one snippet? Less work to copy-paste
```suggestion
# Change to tenant config repo
pushd "inventory/classes/${TENANT_ID}"

# Remove component openshift4-cloudscale and add component openshift4-terraform
yq eval -i '.applications = (.applications[] |
  select(. != "openshift4-cloudscale") | [[.]][]) + (["openshift4-terraform"])' \
  "${CLUSTER_ID}.yml"

# Move parameters from openshift4_cloudscale to openshift4_terraform
yq eval -i '.parameters.openshift4_terraform = .parameters.openshift4_cloudscale |
  del(.parameters.openshift4_cloudscale) |
  .parameters.openshift4_terraform.terraform_variables =
    .parameters.openshift4_terraform.variables |
  del(.parameters.openshift4_terraform.variables)' "${CLUSTER_ID}.yml"
----
```

File: docs/modules/ROOT/pages/how-tos/migrate-from-openshift4-cloudscale.adoc (hunk 2 above)
Comment (simu) [reply docs/modules/ROOT/pages/how-tos/migrate-from-openshift4-cloudscale.adoc]: done

**Review by ccremer — APPROVED** [review]

**Pull request merged**